nevergrad/parametrization/core.py

```diff
@@ -277,10 +277,7 @@ def satisfies_constraints(self) -> bool:
         if not self._constraint_checkers:
             return True
         val = self.value
-        def ok(x: tp.Union[bool, float]) -> bool:
-            """Adapts a constraints (possibly formulated as a float >= 0 or as a bool) to the generic boolean case."""
-            return x >= 0. if isinstance(x, float) else x
```

Comment:
not that this can be dangereous... there are floats, and np.floats, and bools and np.bools.... which may make the behavior impredictible :s but not much we can do about it for now :s

Reply:
We can add "assert isinstance(x, float) or isinstance(x, bool) or isinstance(x, np.bool)" ?

Reply:
(so at least we will know it if something goes wrong)

Reply:
done